File: backend/src/main/java/de/neuefische/backend/model/UnitsList.java
package de.neuefische.backend.model;

public enum UnitsList {
    GRAM("gm(s)"),
    LITER("l(s)"),
    KILOGRAM("kg(s)"),
    MILLILITER("ml(s)"),
    PIECE("pc(s)");

    private final String valueString;
    UnitsList(String valueString) {
        this.valueString = valueString;
    }

    public String getValueString() {
        return valueString;
    }
}

File: frontend/src/UnitsList.ts
// UnitsList.ts
export enum UnitsList {
    GRAM = "GRAM",
    KILOGRAM = "KILOGRAM",
    LITER = "LITER",
    MILLILITER = "MILLILITER",
    PIECE = "PIECE"
}

export const UnitsLabels: Record<UnitsList, string> = {
    [UnitsList.GRAM]: "gm(s)",
    [UnitsList.KILOGRAM]: "kg(s)",
    [UnitsList.LITER]: "l(s)",
    [UnitsList.MILLILITER]: "ml(s)",
    [UnitsList.PIECE]: "pc(s)",
};